internal/experiment/dash/measurer.go

```diff
@@ -125,26 +124,26 @@ func NewExperimentMeasurer(config Config) model.ExperimentMeasurer {
 	return &Measurer{config: config}
 }
 
+var _ model.MeasurementSummaryKeysProvider = &TestKeys{}
+
 // SummaryKeys contains summary keys for this experiment.
-//
-// Note that this structure is part of the ABI contract with ooniprobe
-// therefore we should be careful when changing it.
```

Comment:
We can safely remove this comment because we have moved the implicit requirement that every structure that implements summary MUST contain a field called `IsAnomaly` with boolean type.